internal/pkg/service/common/etcdop/op/txn_test.go

```diff
@@ -363,6 +363,83 @@ func TestTxnOp_ThenTxn_CalledWithoutTxn(t *testing.T) {
 	}
 }
 
+func TestTxnOp_Then_Simple(t *testing.T) {
+	t.Parallel()
```

Comment:
There are already enough tests here, but I added another one to show the difference between ThenTxn and Merge.